agent/command/macsign_test.go

```diff
@@ -0,0 +1,204 @@
+package command
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"path/filepath"
+	"testing"
+
+	"github.com/evergreen-ci/evergreen/agent/internal"
+	"github.com/evergreen-ci/evergreen/agent/internal/client"
+	"github.com/evergreen-ci/evergreen/model"
+	"github.com/evergreen-ci/evergreen/model/task"
+	"github.com/evergreen-ci/evergreen/util"
+	. "github.com/smartystreets/goconvey/convey"
+	"github.com/stretchr/testify/require"
+)
+
+func TestMacSignValidateParams(t *testing.T) {
+
+	Convey("With a mac signing and notarization command", t, func() {
```

Comment:
For future reference (if ever haha) we don't actually use Convey for testing anymore; instead we use assert / require. You don't have to fix this if you don't want to though since it's already done with Convey now.